docs/en/sdk-manual/migrate-from-third-party-sdks.md

```diff
@@ -0,0 +1,122 @@
+# Migrate from third-party SDKs
+
+## Introduction
+
+Generally, in order not to affect your normal business analysis, when migrating from a third-party SDK to Clickstream SDK, we recommend that you integrate your existing SDK with Clickstream SDK for a period of time, and then remove it when you think that Clickstream SDK can replace the previous SDK.
+
+This article mainly introduces how to integrate Clickstream SDK and third-party SDK into your application at the same time. After understanding this article, you will be able to synchronously integrate third-party SDKs with the Clickstream SDK with minimal code modifications, which will help you transition your data analytics business from other SDKs to Clickstream more smoothly.
```

Comment:
In summary, we recommend you create one overarching analytic logger function that encapsulates all the event reporting methods from both legacy SDK and Clickstream SDK, so that you have one API to log event data to multiple destinations. Once you satisfy with the data, you can easily update the function to disable the legacy SDK data reporting.

Reply:
changed to this